### DISCUSSION TITLE
io/serialize

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#251] [Announcements]
# io/serialize

本文详细介绍了 Java 序列流在对象序列化和反序列化中的重要作用，阐述了其如何有效地将 Java 对象持久化存储和恢复。同时，文章还提供了序列流的实际应用示例和常用方法。阅读本文，将帮助您更深入地了解 Java 序列流以及其在 Java 编程中的关键地位，提高数据持久化和恢复的效率。

https://javabetter.cn/io/serialize.html

**Comment by 2in9ya2** [comment]
为啥报错了呢，用Kryo 
java.lang.IllegalAccessError: failed to access class org.example.KryoParam

**Reply by mking2023** [reply]
解决了吗，我也是这个错误

**Comment by 2in9ya2** [comment]
没有

mking2023 ***@***.***> 于 2023年11月6日周一 15:01写道：

**Comment by Knightzh** [comment]
开头的文字和图都搞反了，ObjectInputStream是反序列化，ObjectOutStream才是序列化
